sim_pipeline/RSP.py

```diff
@@ -0,0 +1,128 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import mpl_toolkits.axisartist.floating_axes as floating_axes
+from mpl_toolkits.axisartist.grid_finder import (FixedLocator, MaxNLocator,
+                                                 DictFormatter)
+from matplotlib.transforms import Affine2D
+import pickle
+import lsst.daf.butler as dafButler
+import lsst.geom as geom
+# Source injection
+from lsst.pipe.tasks.insertFakes import _add_fake_sources
+from sim_pipeline.image_simulation import gsobj_true_flux
+import galsim
+from astropy.io import fits
+from astropy.table import Table
+from sim_pipeline.image_simulation import sharp_image
+
+def DC2_cutout(ra, dec, num_pix, butler, band):
```

Comment:
please describe what this function does, also describe a bit more what are required for the butler initialization